.github/workflows/publish-nym-vpn-android.yml

```diff
@@ -101,6 +101,13 @@ jobs:
           name: ${{ env.UPLOAD_DIR_ANDROID }}
           path: ${{ github.workspace }}/temp
 
+      - name: Get checksum
+        id: checksum
+        run: |
+          file_path=$(find ${{ github.workspace }}/temp -type f -iname "*.apk" | tail -n1)
+          echo "checksum=$(apksigner verify -print-certs $file_path | grep -Po "(?<=SHA-256 digest:) .*" | tr -d "[:blank:]")" >> $GITHUB_OUTPUT
+
```

Comment:
Crazy echo command, messing with quotes, is this working as expected?

Reply:
Yes, it works 😆